recipes/biodigest/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "biodigest" %}
+{% set version = "0.2.16" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7618cedf40fedd6dd60aa2fbf8635fb4af0a2c0ae16c01ef9741d30729b6e8bc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('biodigest', max_pin="x.x") }}
+
+requirements:
+  host:
```

Comment:
I think only `python` and `pip` need to be under `host`

Reply:
thank you, I deleted the rest.